libs/komencikit/scripts/proxy-bytecode-sha3.js

```diff
@@ -1,6 +1,11 @@
 const Proxy = require('@celo/protocol/build/contracts/Proxy.json')
+const ProxyV2 = require('@celo/protocol/build/contracts/ProxyV2.json')
 const Web3 = require('web3')
 
 console.log('Constructing stripped bytecode SHA3 for the Proxy contract')
-const bytecode = Proxy.deployedBytecode.split('a265627a7a72315820')[0]
-console.log('Bytecode hash:', Web3.utils.soliditySha3(bytecode))
+const proxyBytecode = Proxy.deployedBytecode.split('a265627a7a72315820')[0]
+console.log('Bytecode hash:', Web3.utils.soliditySha3(proxyBytecode))
+
+console.log('Constructing stripped bytecode SHA3 for the ProxyV2 contract')
```

Comment:
Do we care about the EIP-1167 proxy bytecode at all? Still need to find where this is used, but I thought it was for Valora confirming that the proxy Komenci gave it is valid; if that's the case, won't Valora need to check the EIP-1167 bytecode instead?

Reply:
I'm guessing you figured out my intentions here.